interface-elements-for-web/articles/tree-list/layout-and-navigation/reorder-columns.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Reorder Columns
+author: Bogdan Harchenko
+legacyId: 4532
+---
+# Reorder Columns
+You can drag a column header by mouse and drop it at the required position within the column header panel.
```

Comment:
"drag a column header by mouse" drag by mouse?

Reply:
Why "required"?

Reply:
Perhaps: "You can drag a column header and drop it to another position in the column header panel."